File: Python/섹션 3 - (진행중) 탐색&시뮬레이션(string,1차원,2차원 리스트 탐색)/2. 숫자만 추출 (중상)/AA.py
import sys
#sys.stdin=open('input.txt', 'rt')

s=input()

temp=''
cnt=0
for c in s:
    if c.isdecimal():
        if cnt>0 or int(c)!=0:
            temp+=c
            cnt+=1

arr=[]
temp=int(temp)
for i in range(1, temp+1):
    if temp%i==0:
        arr.append(i)

print(temp)
print(len(arr))

File: Javascript/섹션1. 기본 문제 풀이/15. 가운데 문자 출력 (하)/AA.js
function solution(str){
    let answer=''
    
    let temp=''
    let cnt=0
    for(let x of str){
        cnt++
        if(cnt==1 || cnt==str.length){
            continue
        }
        temp+=x
    }
    answer=temp
    return answer
}
let str="canyoulookatme?"
console.log(solution(str))